src/main/java/xyz/aadev/generitech/common/tileentities/TileEntityMachineBase.java

```diff
@@ -35,8 +35,45 @@
 package xyz.aadev.generitech.common.tileentities;
 
 
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.ITickable;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraft.world.WorldServer;
+import xyz.aadev.aalib.common.network.NetworkExample;
+import xyz.aadev.aalib.common.tileentities.TileEntityBase;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'xyz.aadev.aalib.common.tileentities.TileEntityBase'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)